sources/packages/api/src-sql/EducationPrograms/Create-education-programs.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS education_programs(
+  id SERIAL PRIMARY KEY,
+  program_name VARCHAR(300) NOT NULL,
+  program_description VARCHAR(300) NOT NULL,
+  credential_type VARCHAR(50) NOT NULL,
+  credential_type_other VARCHAR(300) NOT NULL,
+  cip_code VARCHAR(50) NOT NULL,
+  noc_code VARCHAR(50) NULL,
+  sabc_code VARCHAR(50) NULL,
+  regulatory_body VARCHAR(100) NOT NULL,
+  delivered_on_site BIT NOT NULL, 
+  delivered_online BIT NOT NULL,
+  delivered_online_also_onsite VARCHAR(50) NULL, 
+  same_online_credits_earned VARCHAR(50) NULL,
+  earn_academic_credits_other_institution VARCHAR(50) NULL,
+  course_load_calculation VARCHAR(50) NOT NULL,
+  average_hours_study INT NOT NULL,
+  completion_years VARCHAR(50) NOT NULL,
```

Comment:
`year`, will it be better to use INTEGER

Reply:
I feel it is okay as per the UI the values are not integers, it like 

- 12weeksOrLessThen1year
- 2yearsToLessThan3years
- 3yearsToLessThan4years
etc..

Reply:
As per @guru-aot mention, we are going to store something different than a number.